x/oracle/keeper/historic_price.go

```diff
@@ -45,7 +45,7 @@ func (k Keeper) CalcAndSetHistoricMedian(
 
 	median, err := decmath.Median(historicPrices)
 	if err != nil {
-		return errors.Wrap(err, "denom: "+denom) //nolint: goconst
+		return errors.Wrap(err, "denom: "+denom)
```

Comment:
Why did we remove this?

Reply:
the `//nolint` is a directive to ignore linting a line. It's good if we can remove this.